### ISSUE TITLE
[FEATURE REQUEST] Add Kibana reports for Open Distro for Elasticsearch

### ISSUE THREAD

**Issue by rafzei** [epiphany-platform/epiphany#1914]
**Is your feature request related to a problem? Please describe.**
Add reports plugin for Kibana ODFE. 
The release of [kibana-reports (v.1.12)](https://github.com/opendistro-for-elasticsearch/kibana-reports/releases) is tagged as pre-release.
This should be done along with #1859 

**Additional context**
Internal request

**Comment by rpudlowski93** [comment]
At the beginning, I added Kibana reports plugin as standalone Kibana plugin, but after that I created cluster and I noticed that all the kibana plugins are installed by default so my change was not necessary and I removed my code. Generally the task can be closed since the plugin is already installed by default as I mentioned but I was checking how it works and if everything looks fine and I found one problem which looks like a bug: it is not possible to download reports from dashboards or visualizations to PDF/PNG due to error to api - code 404, not found. 
I already reported it to official repository of kibana reports plugin here as a bug: [link](https://github.com/opendistro-for-elasticsearch/kibana-reports/issues/365). I also created separated task with the bug for our purposes: task #2261 

Finally, I would suggest to close the task since the kibana reports plugin is installed by default in our new versions of epiphany where we have ODFE in version 1.13.x and generating CSV files is possible: 
![Screenshot 2021-04-28 at 13.48.28.png](https://images.zenhubusercontent.com/5efee6ff6d0fa19127832e7d/a828a8f3-55fd-4033-9e98-8cc4d5290e14)
